packages/transaction_signing_gateway/lib/storage/flutter_secure_storage_data_store.dart

```diff
@@ -52,4 +52,24 @@ class FlutterSecureStorageDataStore implements SecureDataStore {
       );
     }
   }
+
+  @override
+  Future<Either<CredentialsStorageFailure, bool>> clearAllData() async {
+    try {
+      await _store.deleteAll(
+        iOptions: const IOSOptions(accessibility: IOSAccessibility.passcode),
```

Comment:
let's extract the IOSOptions and AndroidOptions to private static const fields inside this class to keep it DRY

Reply:
will do